### PR TITLE
Issue #312 Enable integration tests on Centos CI

### LINKED ISSUE #312: Enable integration tests on CentOS CI
See also - https://github.com/minishift/minishift/pull/311#discussion_r97033189

### PR DESCRIPTION
**Pull request by budhrg**
Fix #312

### CODE REVIEW

**Comment by budhrg** [comment]
retest this please

**Comment by hferentschik** [comment]
Marking this as WIP until we look into the issue of setting the GitHub access token as well

**Comment by budhrg** [comment]
Centos CI is dependent on it now https://github.com/minishift/minishift-ci-jobs/issues/10

**Comment by budhrg** [comment]
@hferentschik CI is passing https://ci.centos.org/job/minishift/52/console (note I have comment other checks like `make test cross ... prerelease` to reduce API rate limit for testing purpose).

IMO, its ok to merge this PR.

File: centos_ci.sh [hunk 1]
```diff
@@ -66,5 +76,8 @@ mkdir /tmp/glide
 tar --directory=/tmp/glide -xvf glide-${GLIDE_TAG}-${GLIDE_OS_ARCH}.tar.gz
 export PATH=$PATH:/tmp/glide/${GLIDE_OS_ARCH}
 
+make clean
 # Test
 make test cross fmtcheck prerelease
+# Run integration test with 'kvm' driver
+VM_DRIVER=kvm make integration
```
Comment (hferentschik) [inline centos_ci.sh]: So we would just do integration tests on CentOS CI. That's really sub optimal

**Comment by hferentschik** [comment]
Seems build 53 then failed again - https://ci.centos.org/job/minishift/53/ :-(

**Comment by hferentschik** [comment]
I don't think we get around setting the token somewhere, somehow

**Comment by hferentschik** [comment]
retest this please

**Comment by budhrg** [comment]
@hferentschik 

> I don't think we get around setting the token somewhere, somehow

It will be set in CentOS CI jenkins job I believe. See https://github.com/minishift/minishift-ci-jobs/issues/10

**Comment by hferentschik** [comment]
retest this please

**Comment by hferentschik** [comment]
Merged, thanks

**Comment by budhrg** [comment]
@hferentschik The token has been added via PR https://github.com/minishift/minishift-ci-jobs/pull/11 and hope we will not face API rate limit error any more now.

Also, it has been added across minishift ecosystem.

**Comment by hferentschik** [comment]
+1 Nice

**Pull request closed without merging**